modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-an-existing-ingress-controller_{context}"]
+= Modifying an existing Ingress Controller
+
+As a cluster administrator, you can modify an existing Ingress Controller to manage the DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* A DNS record is created and published.
+
+.Procedure
+
+. Modify the custom `IngressController` custom resource to set `dnsManagementPolicy`:
+
++
+[source,terminal]
+----
+oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"dnsManagementPolicy":"Unmanaged", "scope":"<current-scope>"}}}}'
```

Comment:
Do they need to change scope for this?
```suggestion
oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"dnsManagementPolicy":"Unmanaged"}}}}'
```

Reply:
@thejasn ptal

Reply:
They don't need to change the scope. Since scope is a required field, we can't really patch `loadBalancer`, we end up with the validation error. 
```
The IngressController "default" is invalid: spec.endpointPublishingStrategy.loadBalancer.scope: Required value
```

Reply:
I guess we have to tell people how to get the `current-scope` then?

Reply:
@thejasn ptal

Reply:
I've added a [suggestion](https://github.com/openshift/openshift-docs/pull/49752/files#r974248475), let me know if this is sufficient for the end-user.